wemake_python_styleguide/options/validation.py

```diff
@@ -59,6 +59,7 @@ class _ValidatedOptions(object):
     max_asserts: int = attr.ib(validator=[_min_max(min=1)])
     max_access_level: int = attr.ib(validator=[_min_max(min=1)])
     max_attributes: int = attr.ib(validator=[_min_max(min=1)])
+    nested_classes_whitelist: List[str]
```

Comment:
It is a `FrozenSet[str]`, not a list.

Reply:
It was first in the first implementation. But flake8 provide a list of strings. I thought that using `List[str]` type annotation will be closer to reality. But ok: I can change annotation to `FrozenSet[str]`. Do it?